packages/contracts/test/attestation.test.ts

```diff
@@ -347,21 +318,24 @@ describe('Attestations', function () {
             const negRep = Math.floor(Math.random() * 10)
             const graffiti = genRandomSalt()
 
-            {
-                await unirepContract
-                    .connect(attester)
-                    .submitAttestation(
-                        epoch,
-                        epochKey,
-                        posRep,
-                        negRep,
-                        graffiti
-                    )
-                    .then((t) => t.wait())
-            }
+            const receipt = await unirepContract
```

Comment:
Actually this entire thing can be de-structured in one line
```js
const { timestamp } = await unirepContract
  .connect(attester)
  .submitAttestation(epoch, epochKey, posRep, negRep, graffiti)
  .then((t) => t.wait())
  .then(({ blockNumber }) => ethers.provider.getBlock(blockNumber))
```

This way we avoid having too many local variables